packages/client/src/containers/ContactUs.container.js

```diff
@@ -10,8 +10,11 @@ export const ContactUs = () => {
   });
   const [formErrors, setFormErrors] = useState({});
   const [isSubmit, setIsSubmit] = useState(false);
+  const [disabled, setDisabled] = useState(true);
 
   const handleChange = (e) => {
+    setFormErrors({});
+    setIsSubmit(false);
```

Comment:
I don't think this is needed, the form errors will be set once the form is submitted since that's when the validation happens.

Further, the submit flag should be changed to reflect what you actually mean "messageSentSuccessfully".
Since that will depend on the API, for now it's enough to rename it and only set it to true once the form is submitted.